passgithelper.py

```diff
@@ -340,6 +340,8 @@ def get_password(request, mapping) -> None:
         return
 
     host = request['host']
+    if 'username' in request:
+        username = request['username']
     if 'path' in request:
```

Comment:
This looks duplicated because the same condition is used again in the hunk below.

Reply:
> This looks duplicated because the same condition is used again in the hunk below.

Yeah, looks like it. Amended.